app/src/main/java/com/quran/labs/androidquran/util/StorageUtils.java

```diff
@@ -68,35 +78,11 @@
       String firstItem = mounts.iterator().next();
 
       // Follow Android SDCards naming conventions
-      if (Build.VERSION.SDK_INT < Build.VERSION_CODES.GINGERBREAD) {
-        list.add(new Storage(
-            context.getString(R.string.prefs_sdcard_auto),
```

Comment:
this isn't used anymore, can we remove the string from the strings files? speaking of which, we need a utility to make this easier, it's a hassle right now :(